templates/frame/index.html

```diff
@@ -0,0 +1,452 @@
+                        width="156",
+                        height="312",
+                        hi_def=True,
+                        loading="lazy",
+                        attrs={"class": "p-logo-section__logo"}) | safe
+            }}
+          </div>
+          <div class="p-logo-section__item">
+            {{ image(url="https://assets.ubuntu.com/v1/3fa92fbd-flutter-logo.png",
+                        alt="Flutter",
+                        width="138",
+                        height="312",
+                        hi_def=True,
+                        loading="lazy",
+                        attrs={"class": "p-logo-section__logo"}) | safe
+            }}
+          </div>
+          <div class="p-logo-section__item">
+            {{ image(url="https://assets.ubuntu.com/v1/f595e3d7-react-logo.png",
+                        alt="Electron",
+                        width="144",
+                        height="312",
+                        hi_def=True,
+                        loading="lazy",
+                        attrs={"class": "p-logo-section__logo"}) | safe
+            }}
+          </div>
+          <div class="p-logo-section__item">
+            {{ image(url="https://assets.ubuntu.com/v1/3bf2462f-GTK-logo.png",
+                        alt="GTK",
+                        width="252",
+                        height="312",
+                        hi_def=True,
+                        loading="lazy",
+                        attrs={"class": "p-logo-section__logo"}) | safe
+            }}
+          </div>
+          <div class="p-logo-section__item">
+            {{ image(url="https://assets.ubuntu.com/v1/f2f331ca-html5-logo.png",
+                        alt="HTML5",
+                        width="43",
+                        height="104",
+                        hi_def=True,
+                        loading="lazy",
+                        attrs={"class": "p-logo-section__logo"}) | safe
+            }}
+          </div>
+          <div class="p-logo-section__item">
+            {{ image(url="https://assets.ubuntu.com/v1/bcdedc4b-sdl-logo.png",
+                        alt="Simple Directmedia Layer",
+                        width="198",
+                        height="312",
+                        hi_def=True,
+                        attrs={"class": "p-logo-section__logo"}) | safe
+            }}
+          </div>
+        </div>
+      </div>
+    </div>
+  </section>
+
+  <section class="p-section">
+    <hr class="p-rule is-fixed-width" />
+    <div class="grid-row">
+      <div class="grid-col-small-8 grid-col-medium-4 grid-col-4">
+        <h2>
+          One tool,
+          <br class="u-hide--small" />
+          all key interaction functionalities
+        </h2>
+      </div>
+      <div class="grid-col-small-8 grid-col-medium-4 grid-col-4">
+        <video autoplay=""
+               loop=""
+               muted=""
+               class="p-section--shallow u-no-margin--bottom">
```

Comment:
```suggestion
       <div class="p-section--shallow">
        <video autoplay=""
               loop=""
               muted=""
               class="p-section--shallow u-no-margin--bottom">
        </div>
```
I would recommend separating spacing elements.